src/types/Comment.ts

```diff
@@ -1,6 +1,13 @@
 export interface Comment {
   id: number;
-  postId: number;
+  postId: number | undefined;
```

Comment:
It's not recommended to use undefined as a type for postId. If a postId is optional, use a question mark instead.